modules/manage/pages/manage-xdcr/create-xdcr-reference.adoc

```diff
@@ -81,7 +83,8 @@ image::manage-xdcr/xdcr-add-remote-cluster-dialog.png[,400,align=left]
 +
 The fields in this dialog are explained above, in xref:manage:manage-xdcr/create-xdcr-reference.adoc#understanding-references[Understanding References].
 
-. For *Cluster Name* and *IP/Hostname*, specify the IP address of the second cluster, which is `10.144.210.102`.
+. For *Cluster Name*, provide a name for the target cluster: this name is for use on the _source_ cluster only, and so need not be identical to any name defined on the _target_ cluster.
+For *IP/Hostname*, specify the IP address (if appropriate, the _alternate address_) or the hostname of the target cluster: in this case, the IP address to be used is `10.144.210.102`.
```

Comment:
In the context of XDCR, it is probably better to replace "hostname" with "FQDN" since they may not be in the same domain.

Reply:
Hostname is still valid - so maybe hostname or FQDN

Reply:
I've changed it to: For IP/Hostname, specify the IP address (if appropriate, the alternate address), the hostname, or the fully qualified domain-name of the target cluster: in this case, the IP address 10.144.210.102 is to be used.